Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -652,7 +657,9 @@ void NativeReanimatedModule::performOperations() {
 
           return newRoot;
         },
-        {/* default commit options */});
+        {.shouldYield = [this]() {
+          return propsRegistry_->shouldSkipCommit();
+        }});
```

Comment:
Looks like `.foo =` is a feature from C++20 but we need to support C++17, [here's](https://github.com/facebook/react-native/blob/3e828691ddbb909190f33b562fd0ba3c0f3df23a/packages/react-native/ReactCommon/react/renderer/uimanager/UIManager.cpp#L98) how it looks in react-native, maybe we should use the same technique here?
```suggestion
        {/* .shouldYield = */ [this]() {
          return propsRegistry_->shouldSkipCommit();
        }});
```

Reply:
ok